export const signUp = async ()=>{
    try{


    }catch(error){
        console.log(`Error :- ${error}`);
    }
}


export const login = async ()=>{
    try{

            
    }catch(error){
        console.log(`Error :- ${error}`);
    }
}

export const logout = async ()=>{
    try{

            
    }catch(error){
        console.log(`Error :- ${error}`);
    }
}